lib/dgram.js

```diff
@@ -97,19 +100,24 @@ function newHandle(type, lookup) {
   throw new ERR_SOCKET_BAD_TYPE();
 }
 
-
 function _createSocketHandle(address, port, addressType, fd, flags) {
-  // Opening an existing fd is not supported for UDP handles.
-  assert(typeof fd !== 'number' || fd < 0);
-
-  var handle = newHandle(addressType);
+  const handle = newHandle(addressType);
+  let err;
 
-  if (port || address) {
-    var err = handle.bind(address, port || 0, flags);
-    if (err) {
-      handle.close();
-      return err;
+  if (isUint32(fd)) {
```

Comment:
Small detail, but I don't think we want a uint32, but rather an int32 that is not negative.